### PR TITLE
Add a Dockerfile and k8s cron job to upload files to GCS each night.

### PR DESCRIPTION
**Pull request by dlorenc**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM golang
```
Comment (inferno-chromium) [inline Dockerfile:1]: nit: probably need the license header in this and cron.sh files.

File: cron/cron.sh [hunk 2]
```diff
@@ -9,3 +9,5 @@ do
     echo $line
     go run . --repo=$line --format=csv 2>/dev/null >> $output
```
Comment (inferno-chromium) [inline cron/cron.sh:10]: should we build it once, and then just run ./scorecard binary.

File: cron/cron.sh [hunk 3]
```diff
@@ -9,3 +9,5 @@ do
...
 done < "$input"
+
+gsutil cp $output gs://ossf-scorecards/
```
Comment (inferno-chromium) [inline cron/cron.sh:13]: maybe gs://ossf-scorecards/ should come from ENV var, that ENV var can be set in Dockerfile,

ENV SCORECARS_BUCKET gs://ossf-scorecards

**Review by inferno-chromium — APPROVED** [review]

File: cron/cron.sh (hunk 2 above)
Comment (dlorenc) [reply cron/cron.sh:10]: Fixed up the Dockerfile into a proper multi-stage one.

File: cron/cron.sh (hunk 3 above)
Comment (dlorenc) [reply cron/cron.sh:13]: handled this in the k8s yaml instead, it's basically the same thing

**Review by inferno-chromium — APPROVED** [review]

**Pull request merged**